.github/workflows/actions.yml

```diff
@@ -501,6 +501,67 @@ jobs:
           name: source-tarball
           path: obj/transmission*.tar.*
 
+  macos-14-universal-from-tarball:
```

Comment:
Why this is being built from tarball and not from checkout though?

Reply:
Doesn't matter I think. Just, the only configuration with `USE_SYSTEM_*=OFF` was for the tarball job previously, so I kept it like that.